index.bs

```diff
@@ -253,18 +253,251 @@ Control Protocols {#control}
+connections, thus making message demuxing/routing easier).
+
+
+A controller may terminate a connection without terminating the presentation by
+sending a presentation-connection-close-request message with the following
+values:
+
+- connection-id: The ID of the connection to close.
+
+
+The receiver should, upon receipt of a presentation-connection-close-request,
+send back a presentation-connection-close-response message with the following
+values:
+
+- result: If the close succeed or failed, and if it failed why it failed.
+
+The receiver may also close a connection without a request from the controller
+to do so and without terminating a presentation.  If it does so, it should send
+a presentation-connection-close-event to the controller with the following
+values:
+
+- connection-id: The ID of the connection that was closed
+
+- reason: The reason the connection was closed
+
+- error-message: A debug message suitable for a log or perhaps presented to the
+user with more explanation as to why it was closed.
+
+<!-- TODO: Why does the Presentation API spec not mention the use of the close
+message? -->
+
+<!-- TODO: Specify message ordering groups. -->
+
+
+Presentation API {#presentation-api}
+---------------------------------------------
+
+This section defines how [[PRESENTATION-API|Presentation API]] uses
+the messages defined in the previous sections.  
+Non-browser endpoints can also send and receive the same messages defined here.
+If so, a non-browser endpoint must follow the same restrictions for the
+presentation-id as the a does, as defined by [[PRESENTATION-API|Presentation
+API]] section 6.1 (at least 16 ASCII characters).  
+
+When [[PRESENTATION-API|Presentation API]] [section
+6.4.2](https://www.w3.org/TR/presentation-api/#sending-a-message-through-presentationconnection)
+says "This list of presentation displays ... is populated based on an
+implementation specific discovery mechanism", the [=controlling user
+agent=] may use the mDNS, QUIC, endpoint-info-request, and
+presentation-url-availability-request messages defined previously in
+this spec to discover receivers.
+
+When [[PRESENTATION-API|Presentation API]] [section
+6.4.2](https://www.w3.org/TR/presentation-api/#the-list-of-available-presentation-displays)
+says "To further save power, ... implementation specific discovery of
+presentation displays can be resumed or suspended.", the [=controlling
+user agent=] may use the power saving mechanism defined in the
+previous section.
+
+When [[PRESENTATION-API|Presentation API]] [section
+6.3.4](https://www.w3.org/TR/presentation-api/#starting-a-presentation-connection)
+says "Using an implementation specific mechanism, tell U to create a
+receiving browsing context with D, presentationUrl, and I as
+parameters.", U (the [=controlling user agent=]) may send a
+presentation-start-request message to D (the receiver), with I for the
+presentation identifier and presentationUrl for the selected
+presentation URL.
+
```

Comment:
Adding a section for reconnect would be good.  The spec describes reconnecting to presentations within the same browser, but leaves undefined reconnecting to presentations that are running outside the browser.  I can add a note to the spec that reconnecting to presentations on other displays is an implementation specific detail too :-)

Reply:
If you add text that I can quote, then I'll quote it and link to it.  

I'll put a TODO for now